src/onnx/onnx.cpp

```diff
@@ -1067,6 +1068,187 @@ struct onnx_parser
         return prog.add_instruction(op, make_contiguous(args[0]));
     }
 
+    const auto& get_nearest_op(const std::string& mode)
```

Comment:
These functions can be declared `static`, I believe? I dont think they need to be a member function.

Reply:
fixed